### PR TITLE
feat: Remove old schemas and re-version all schemas and protobufs to v1

### LINKED ISSUE #189: Move old schemas to new repo, re-version all schemas to v1
we should create a new repo ([i.e. @mapeo/schema-legacy](https://github.com/digidem/schema-legacy)) and move all the schemas on `old/` to that repo.
additionally we should change all version (through file renaming) to v1 before MPV. This versioning change wouldn't be a breaking change since we always use the last version for validation and encoding

### PR DESCRIPTION
**Pull request by tomasciccola**
Should close #189

### CODE REVIEW

File: proto/projectSettings/v1.proto [hunk 1]
```diff
@@ -5,6 +5,8 @@ import "google/protobuf/timestamp.proto";
 import "common/v1.proto";
 import "options.proto";
 
+// Same as v1, created to test forward compatibility
+
```
Comment (EvanHahn) [inline proto/projectSettings/v1.proto:9]: I don't understand this comment. Should we remove it?

```suggestion
```

**Review by EvanHahn — APPROVED** [review]

File: proto/projectSettings/v1.proto (hunk 1 above)
Comment (tomasciccola) [reply proto/projectSettings/v1.proto]: projectSettings was copied (with the same content) to test forward compatibility. It doesn't make sense to keep it now...

**Comment by tomasciccola** [comment]
I'm wondering what work is needed left for bumping this to a major version (v1). Looking at the release.yml I see that `prerelease` is publish under `next` dist-tag, but this release would be a major version so I'm wondering if the release flow would take care of this...

**Comment by tomasciccola** [comment]
I'm going to go ahead and merge it. We can address my comment later

**Comment by EvanHahn** [comment]
Yeah, #371 is a breaking change that would require a major version. I think we should wait until that's done.

**Pull request merged**